### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by hieubm**

### CODE REVIEW

File: README.rdoc [hunk 1]
```diff
@@ -1,28 +1,2 @@
...
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
-
-
-Please feel free to use a different markup language if you do not plan to run
-<tt>rake doc:app</tt>.
+#Smart Octocat
```
Comment (NeverSmileK57CLC) [inline README.rdoc]: :-1:

Comment (NeverSmileK57CLC) [inline README.rdoc]: abc

File: README.rdoc [hunk 2]
```diff
@@ -1,28 +1,2 @@
-== README
-
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
-* Ruby version
-
```
Comment (NeverSmileK57CLC) [inline README.rdoc]: hehe

File: README.rdoc [hunk 3]
```diff
@@ -1,28 +1,2 @@
...
-* System dependencies
-
-* Configuration
-
-* Database creation
```
Comment (NeverSmileK57CLC) [inline README.rdoc]: hoho

File: README.rdoc [hunk 4]
```diff
@@ -1,28 +1,2 @@
...
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
```
Comment (NeverSmileK57CLC) [inline README.rdoc]: hihi

File: README.rdoc [hunk 5]
```diff
@@ -1,28 +1,2 @@
...
-
-* Deployment instructions
-
```
Comment (NeverSmileK57CLC) [inline README.rdoc]: huhu

File: README.rdoc [hunk 6]
```diff
@@ -1,28 +1,2 @@
...
-* ...
-
```
Comment (NeverSmileK57CLC) [inline README.rdoc]: ooooooo

File: README.rdoc [hunk 7]
```diff
@@ -1,28 +1,2 @@
...
-
```
Comment (NeverSmileK57CLC) [inline README.rdoc]: pppppppppppp

File: README.rdoc [hunk 8]
```diff
@@ -1,28 +1,2 @@
-== README
-
```
Comment (NeverSmileK57CLC) [inline README.rdoc]: abc

File: README.rdoc (hunk 7 above)
Comment (NeverSmileK57CLC) [inline README.rdoc]: ngon

File: README.rdoc (hunk 8 above)
Comment (NeverSmileK57CLC) [inline README.rdoc]: good

Comment (NeverSmileK57CLC) [inline README.rdoc]: 1

Comment (NeverSmileK57CLC) [inline README.rdoc]: 2

Comment (NeverSmileK57CLC) [inline README.rdoc]: 3

Comment (NeverSmileK57CLC) [inline README.rdoc]: 5

Comment (NeverSmileK57CLC) [inline README.rdoc]: Lood good

File: README.rdoc (hunk 1 above)
Comment (NeverSmileK57CLC) [inline README.rdoc]: sdfsd

Comment (NeverSmileK57CLC) [inline README.rdoc]: abcfsdf

Comment (NeverSmileK57CLC) [inline README.rdoc]: sdfsdf

File: README.rdoc (hunk 8 above)
Comment (truonglocbinh) [inline README.rdoc]: So good

File: README.rdoc (hunk 7 above)
Comment (truonglocbinh) [inline README.rdoc]: xzxczxzcxz

Comment (truonglocbinh) [inline README.rdoc]: xzzxzxzx

Comment (truonglocbinh) [inline README.rdoc]: xzzxzx

Comment (truonglocbinh) [inline README.rdoc]: xccxcx

Comment (truonglocbinh) [inline README.rdoc]: cxxcxccx

File: README.rdoc (hunk 8 above)
Comment (NeverSmileK57CLC) [inline README.rdoc]: def

File: README.rdoc (hunk 1 above)
Comment (NeverSmileK57CLC) [inline README.rdoc]: dfg

Comment (NeverSmileK57CLC) [inline README.rdoc]: sdfsd

File: README.rdoc (hunk 8 above)
Comment (NeverSmileK57CLC) [inline README.rdoc]: xcvxcv

Comment (NeverSmileK57CLC) [inline README.rdoc]: 12312323

Comment (NeverSmileK57CLC) [inline README.rdoc]: 123123123

File: README.rdoc (hunk 2 above)
Comment (NeverSmileK57CLC) [inline README.rdoc]: 123123123

File: README.rdoc (hunk 3 above)
Comment (NeverSmileK57CLC) [inline README.rdoc]: 123123

**Comment by NeverSmileK57CLC** [comment]
OK

**Comment by NeverSmileK57CLC** [comment]
KO

File: README.rdoc [hunk 9]
```diff
@@ -1,28 +1,2 @@
-== README
```
Comment (NeverSmileK57CLC) [inline README.rdoc]: abc

Comment (NeverSmileK57CLC) [inline README.rdoc]: asdasd

Comment (NeverSmileK57CLC) [inline README.rdoc]: dfdf

File: README.rdoc (hunk 1 above)
Comment (NeverSmileK57CLC) [inline README.rdoc]: sdf

File: README.rdoc (hunk 8 above)
Comment (NeverSmileK57CLC) [inline README.rdoc]: sdfsdfsdfs

File: README.rdoc (hunk 9 above)
Comment (NeverSmileK57CLC) [inline README.rdoc]: abcdsadf

Comment (NeverSmileK57CLC) [inline README.rdoc]: asdfsad

Comment (NeverSmileK57CLC) [inline README.rdoc]: sdfsdf

Comment (NeverSmileK57CLC) [inline README.rdoc]: fsdfsd

**Comment by NeverSmileK57CLC** [comment]
OK

**Comment by NeverSmileK57CLC** [comment]
OK

**Comment by NeverSmileK57CLC** [comment]
OK

**Comment by NeverSmileK57CLC** [comment]
OK

**Comment by NeverSmileK57CLC** [comment]
OK

File: README.rdoc (hunk 9 above)
Comment (NeverSmileK57CLC) [inline README.rdoc]: kjsdhfsjkd

Comment (NeverSmileK57CLC) [inline README.rdoc]: sdfsdf

**Comment by NeverSmileK57CLC** [comment]
OK

**Comment by NeverSmileK57CLC** [comment]
OK

File: README.rdoc (hunk 1 above)
Comment (NeverSmileK57CLC) [inline README.rdoc]: sdfsd

Comment (NeverSmileK57CLC) [inline README.rdoc]: sdfsd

File: README.rdoc (hunk 9 above)
Comment (NeverSmileK57CLC) [inline README.rdoc]: asd

**Comment by NeverSmileK57CLC** [comment]
sdfsdf

**Comment by NeverSmileK57CLC** [comment]
OkIe

**Comment by NeverSmileK57CLC** [comment]
OK

**Comment by NeverSmileK57CLC** [comment]
OK

File: README.rdoc (hunk 9 above)
Comment (NeverSmileK57CLC) [inline README.rdoc]: ABC

Comment (NeverSmileK57CLC) [inline README.rdoc]: sdf

Comment (NeverSmileK57CLC) [inline README.rdoc]: asdfasdfa

Comment (NeverSmileK57CLC) [inline README.rdoc]: sdfsdf

Comment (NeverSmileK57CLC) [inline README.rdoc]: aslkjdf

**Pull request closed without merging**